testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/assertions/DecisionInstanceAssert.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.process.test.api.assertions;
+
+import java.util.Map;
+
+/** The assertion object to verify a decision. */
+public interface DecisionInstanceAssert {
+  /**
+   * Verifies that the decision is evaluated. The verification fails if the decision is not
+   * evaluated or if the evaluation fails.
+   *
+   * <p>The assertion waits until the decision is evaluated.
+   *
+   * @return the assertion object
+   */
+  DecisionInstanceAssert isEvaluated();
+
+  /**
+   * Verifies that the decision has the given name.
+   *
+   * @param expectedDefinitionName the expected name for this decision
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasName(final String expectedDefinitionName);
+
+  /**
+   * Verifies that the definition has the given definition id.
+   *
+   * @param expectedDefinitionId the expected id for this decision
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasId(final String expectedDefinitionId);
+
+  /**
+   * Verifies that the definition has the given version.
+   *
+   * @param expectedVersion the expected version for this decision
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasVersion(final int expectedVersion);
+
+  /**
+   * Verifies that the decision is evaluated with the expected output. The verification fails if the
+   * decision is not evaluated or the output does not match.
+   *
+   * @param expectedOutputValue the expected output value
+   * @return the assertion object
+   */
+  DecisionInstanceAssert containsOutput(final String expectedOutputValue);
+
+  /**
+   * Verifies that the decision is evaluated with the expected output. The verification fails if the
+   * decision is not evaluated or the output does not match.
+   *
+   * @param expectedOutput the expected output values represented as a key-value map
+   * @return the assertion object
+   */
+  DecisionInstanceAssert hasOutput(final Map<String, Object> expectedOutput);
```

Comment:
:x: The decision output is not always a map object, but depends on the decision type, the hit policy, and the number of outputs. 

For example, a decision table with one output returns a single value, or a list if multiple rules match.  

We should change the type to `Object`.